doc/sphinx-guides/source/developers/dev-environment.rst

```diff
@@ -333,6 +333,218 @@ If you've reconfigured from EZID to DataCite and are seeing ``Response code: 400
 
 ``asadmin create-jvm-options '-Ddataverse.siteUrl=http\://demo.dataverse.org'``
 
+OpenShift
+---------
+
+From the Dataverse perspective, we are in the business of providing a "template" for OpenShift that describes how the various components we build our application on (Glassfish, PostgreSQL, Solr, the Dataverse war file itself, etc.) work together. We publish Docker images to DockerHub at https://hub.docker.com/u/iqss/ that are used in the OpenShift template. Dataverse's use of Docker is documented below in a separate section.
+
+
+The OpenShift template for Dataverse can be found at ``conf/openshift/openshift.json`` and if you need to hack on the template or related files under ``conf/docker`` it is recommended that you iterate on them using Minishift.
+
+Install Minishift
+~~~~~~~~~~~~~~~~~
+
+Minishift requires a hypervisor and since we already use VirtualBox for Vagrant, you should install VirtualBox from http://virtualbox.org .
+
+Download the Minishift tarball from https://docs.openshift.org/latest/minishift/getting-started/installing.html and put the ``minishift`` binary in ``/usr/local/bin`` or somewhere in your ``$PATH``. This assumes Mac or Linux.
+
+At this point, you might want to consider going through the Minishift quickstart to get oriented: https://docs.openshift.org/latest/minishift/getting-started/quickstart.html
+
+Start Minishift
+~~~~~~~~~~~~~~~
+
+``minishift start --vm-driver=virtualbox``
+
+Make the OpenShift Client Binary (oc) Executable
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``eval $(minishift oc-env)``
+
+Log in to Minishift
+~~~~~~~~~~~~~~~~~~~
+
+Note that if you just installed Minishift, you are probably logged in already, but it doesn't hurt to log in again.
+
+``oc login --username developer --password=whatever``
+
+Use "developer" as the username and a couple characters as the password.
+
+Allow Containers to Run as Root in Minishift
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Whoops! This whole section can be removed. The containers run as non-root as of b84526c